CHANGELOG.md

```diff
@@ -15,12 +15,20 @@ You can find its changes [documented below](#010-2024-11-20).
 
 This release has an [MSRV][] of 1.82.
 
+### Added
+
+* Add `BLACK`, `WHITE`, and `TRANSPARENT` constants to the color types. ([#64][] by [@waywardmonkeys][])
```

Comment:
Is it worth noting that this is a breaking change?

It feels like there's an interesting thing here, that ideally we'd have different definitions of breaking for creators of custom colorspaces and users of the crate - creators of custom colorspaces are *so* rare that kind of mad.

I guess color would probably be a good use-case for the semver-trick.

Reply:
There will be some other breaking changes as well. But I guess we could call out that `ColorSpace` must have a `WHITE_COMPONENTS` const now?

Reply:
Yeah, that is the breaking change. Our standards for marking breaking changes aren't very developed.